monte_carlo.py

```diff
@@ -158,11 +167,12 @@ def transport_neutron(mat, bounds, tallies, fission_banks, first_round, mesh):
             mesh.flux_add(cell, tempd)
             
             # nudge neutron and find cell
-            neutron.move(-TINY_MOVE)
+            #neutron.move(-TINY_MOVE)
 
         # check interaction
         if neutron.alive:
-            neutron_interaction = sample_interaction(mat)
+            neutron_interaction = sample_interaction( \
+                    materials[mesh.get_material(neutron.get_cell())])
```

Comment:
I think this would read better if we split this into:
```
mat = materials[mesh.get_material(neutron.get_cell())]
neutron_interaction = sample_interaction(mat)
```